langchain4j-hugging-face/pom.xml

```diff
@@ -72,6 +72,17 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-validator</groupId>
```

Comment:
Why is this required? Please check https://github.com/langchain4j/langchain4j/blob/main/CONTRIBUTING.md

Reply:
that's for url validation. I have run `mvn dependency:analyze` - it's OK.

Reply:
dependency is removed